examples/wallet/app/components/SendTransaction.js

```diff
@@ -29,7 +29,8 @@ export default ({ balance, nodeSettings, sendTransaction }: Props) => {
   const [amount, setAmount] = useState<?number>();
```

Comment:
There are no errors in the source code of this component, but when the wallet is executed the send button could have margin right

Reply:
![feedback](https://user-images.githubusercontent.com/57267331/72281852-1ce46080-361a-11ea-8e92-d524c4f1139a.png)

Reply:
enzo pointed this out already, I'll address it [after testing the form](https://github.com/input-output-hk/js-chain-libs/pull/116/)